examples/reference-entity-model/steps/mapping/mapCustomersWithRelatedEntitiesXML.step.json

```diff
@@ -0,0 +1,177 @@
+
+{
```

Comment:
To keep this as easy to understand as possible, let's remove all the unnecessary properties here - additionalCollections, batchSize, validateEntity, headers, interceptors, provenance, customHook. Doing so makes it easier to see the stuff that matters.